bitwarden_license/src/Sso/Controllers/AccountController.cs

```diff
@@ -183,6 +183,12 @@ public async Task<IActionResult> Login(string returnUrl)
             }
                 
             var ssoToken = context.Parameters[SsoToken.TokenName];
+
+            if(string.IsNullOrWhiteSpace(ssoToken))
```

Comment:
This will likely fail linting.
```suggestion
            if (string.IsNullOrWhiteSpace(ssoToken))
```